src/components/Modals/RenameFile.js

```diff
@@ -1,13 +1,11 @@
-import React, { useCallback } from "react";
```

Comment:
❓ where does this get used?! I didn't think we could rename files?

Reply:
You can but it's a little bit hidden away - if you go to the three dots next to a file in the sidebar (that isn't `main.py`), rename file is in that dropdown menu